packages/fiber/src/web/index.tsx

```diff
@@ -64,7 +64,7 @@ function render<TCanvas extends Element>(
     // Check pixelratio
     if (props.dpr !== undefined && !is.equ(lastProps.dpr, props.dpr)) state.setDpr(props.dpr)
     // Check size
-    if (size !== undefined && !is.equ(lastProps.size, size)) state.setSize(size.width, size.height)
+    if (!is.equ(lastProps.size, size)) state.setSize(size.width, size.height)
```

Comment:
`size` can never be `undefined` because it checked at line 47 and will be set to a default value and thus defined.